google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/ManagedVmServerConfiguration.java

```diff
@@ -63,7 +67,18 @@ public void setGoogleUserName(String googleUserName) {
     this.googleUserName = googleUserName;
   }
 
-  protected EventBus getProjectNameListener() {
-    return projectNameListener;
+  protected void setProjectNameListener(PropertyChangeListener listener) {
+    this.projectNameListener = listener;
+  }
+
+  protected void fireNameChangeEvent(String name) {
+    if(projectNameListener != null) {
+      projectNameListener.propertyChange(new PropertyChangeEvent(
+          this,
+          "cloudProjectName",
+          name, // We don't care about the old name
```

Comment:
we may as well obey the contract, it doesn't seem like a lot of trouble. Maybe I'm OCD.